addons/account/models/account_move.py

```diff
@@ -1694,6 +1694,9 @@ def _get_tax_cash_basis_base_account(self, line, tax):
         '''
         return line.account_id
 
+    def _get_amount_tax_cash_basis(self, amount, line):
+        return line.company_id.currency_id.round(amount)
```

Comment:
> ```python
> def _get_amount_tax_cash_basis(self, percentage_before_rec, line):
>                     percentage_before = percentage_before_rec[line.move.id]
>                     percentage_after = line._get_matched_percentage()[line.move.id]
>                     #amount is the current cash_basis amount minus the one before the reconciliation
>                     amount = line.balance * percentage_after - line.balance * percentage_before
>                     rounded_amt = line.company_id.currency_id.round(amount)
>                     return amount, rounded_amt
> ...
> amount, rounded_amt = self._get_amount_tax_cash_basis(percentage_before_rec, line)
> ```

Reply:
I don't understand why you do not simply do something like:
```python
def _tax_cash_amount(line, percentage_before, percentage_after):
    return line.balance * percentage_after - line.balance * percentage_before

....

amount = self._tax_cash_amount(line, percentage_before, percentage_after)
```

Reply:
Because @hbto doesn't need change `amount` variable
This value is right as it is.

The value that is required to change is the final amount (`rounded_amt` in this case) since that amount is used for compute other values and rounded_amt other ones below of the code.

Reply:
OK, then please update the commit with the appropriate comments, because as it is, it is really unclear why we override the rounding part instead of the amount computation itself.